blocks/search-results/search-results.js

```diff
@@ -40,7 +40,7 @@ function setResultValue(el, value, term) {
 }
 
 function formatSearchResultCount(num, placeholders, term, lang) {
-  if (lang === 'ja') {
+  if (lang === 'jp') {
```

Comment:
Can these `if` be merged?

Reply:
As in `lang === 'ja' || lang === 'jp'` ?